beater/codec/blob_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codec
+
+import (
+	"strings"
+
+	"testing"
+)
+
+func TestBlobCodecNextErr(t *testing.T) {
+	cases := map[string]struct {
+		Data      string
+		Length    int
```

Comment:
Consider renaming to `ExpectLength` or `ExpectedLength` to make it clear that this field is an expected value and is not used as an input for the test.